docs/index.yml

```diff
@@ -1,111 +1,132 @@
-          - text: PowerPoint add-in quick start
-            url: quickstarts/powerpoint-quickstart.md
-          - text: Project add-in quick start
-            url: quickstarts/project-quickstart.md
-          - text: Word add-in quick start
-            url: quickstarts/word-quickstart.md
-          - text: Single sign-on (SSO) quick start
-            url: quickstarts/sso-quickstart.md
-
-  # Card
-  - title: Tutorials
-    linkLists:
-      - linkListType: tutorial
-        links:
-          - text: Excel add-in tutorial
-            url: tutorials/excel-tutorial.md
-          - text: Excel custom functions tutorial
-            url: tutorials/excel-tutorial-create-custom-functions.md
-          - text: Outlook add-in tutorial
-            url: tutorials/outlook-tutorial.md
-          - text: PowerPoint add-in tutorial
-            url: tutorials/powerpoint-tutorial.md
-          - text: Word add-in tutorial
-            url: tutorials/word-tutorial.md
-
-   # Card
-  - title: Resources
-    linkLists:
-      - linkListType: reference
-        links:
-          - text: Ask questions
-            url: https://stackoverflow.com/questions/tagged/office-js
-          - text: Report issues
-            url: https://github.com/officedev/office-js/issues
-          - text: Request features
-            url: https://officespdev.uservoice.com/
-          - text: COM or VSTO add-ins documentation
-            url: /office/client-developer/office-client-development
-          - text: Office Add-ins PnP samples repo
-            url: https://github.com/OfficeDev/PnP-OfficeAddins
-          - text: Office Add-ins blogs
-            url: https://developer.microsoft.com/office/gallery/?filterBy=Add-ins,Blog%20Posts
-          - text: Additional resources
-            url: resources/resources-links-help.md
-            
+    # Card 3
+    - title: Test and debug
+      links:
+        - url: testing/test-debug-office-add-ins.md
+          itemType: overview
+          text: Check out the debugging overview
+        - url: testing/testing-and-troubleshooting.md
+          itemType: concept
+          text: Troubleshoot user errors
+        - url: testing/troubleshoot-manifest.md
+          itemType: architecture
+          text: Verify an add-in's manifest
+    # Card 4
+    - title: Deploy
+      links:
+        - url: publish/publish.md
+          itemType: deploy
+          text: Explore deployment options 
+        - url: publish/centralized-deployment.md
+          itemType: deploy
+          text: Learn about Centralized Deployment
+    # Card 5
+    - title: Publish
+      links:
+        - url: https://docs.microsoft.com/office/dev/store/
+          itemType: overview
+          text: Validate your Office Add-in
+        - url: https://docs.microsoft.com/office/dev/store/add-in-submission-guide
+          itemType: how-to-guide
+          text: Review the Microsoft 365 App Store submission guide
+    # Card 6
```

Comment:
Would it be worth adding a link to the office-js repo for bug reporting?

Reply:
Probably yes